src/psbt.js

```diff
@@ -136,7 +136,7 @@ class Psbt {
       } catch (_) {}
       return {
         script: (0, bufferutils_1.cloneBuffer)(output.script),
-        value: output.value,
+        value: Number(output.value),
```

Comment:
losing precision here

Reply:
not supporting psbt

Reply:
in that case better throw an exception than to produce an invalid result